backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java

```diff
@@ -0,0 +1,60 @@
+package ai.verta.modeldb.common;
+
+import com.google.rpc.Code;
+import com.google.rpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.grpc.protobuf.StatusProto;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class CommonUtils {
+  private static final Logger LOGGER = LogManager.getLogger(CommonUtils.class);
+
+  public static Integer getRegisteredBackgroundUtilsCount() {
```

Comment:
If you have get this here then related all function like `setRegisteredBackgroundUtilsCount` etc should be here in common module.

Reply:
moved